### PR TITLE
Pr/5780

### PR DESCRIPTION
**Pull request by mmpagani**
This PR supersedes https://github.com/gem/oq-engine/pull/5780

### CODE REVIEW

File: openquake/hazardlib/gsim/zalachoris_rathje_2019.py [hunk 1]
```diff
@@ -0,0 +1,404 @@
...
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with OpenQuake. If not, see <http://www.gnu.org/licenses/>.
+
+"""
+Module exports :class:'ZalachorisRathje2019'
+"""
+from __future__ import division
```
Comment (micheles) [inline openquake/hazardlib/gsim/zalachoris_rathje_2019.py:22]: This is not needed anymore

File: openquake/hazardlib/gsim/zalachoris_rathje_2019.py [hunk 2]
```diff
@@ -0,0 +1,404 @@
...
+    DEFINED_FOR_TECTONIC_REGION_TYPE = const.TRT.INDUCED
+
+    #: Supported intensity measure types are spectral acceleration,
+    #: peak ground velocity and peak ground acceleration
+    DEFINED_FOR_INTENSITY_MEASURE_TYPES = set([
+        PGA,
+        PGV,
+        SA
+    ])
+
```
Comment (micheles) [inline openquake/hazardlib/gsim/zalachoris_rathje_2019.py:51]: We are using the set syntax everywhere now, i.e. `DEFINED_FOR_INTENSITY_MEASURE_TYPES = {PGA. PGV, SA}`. Same for similar definitions below.

**Comment by micheles** [comment]
Apart the minor suggestions, LGTM

**Comment by rcgee** [comment]
LGTM

**Pull request merged**